pkg/manifests/extension_policy.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: gateway.envoyproxy.io/v1alpha1
```

Comment:
This still exists in `ext_proc.yaml`, do we need it in both places?

Reply:
It's not required for EPP, it's an Envoy Gateway requirement. I forgot to remove it so thanks for the heads-up. Commit 8ebe6ed5db225549fd782eb35060cfa767a2b76f removes the duplicate EnvoyExtensionPolicy, PTAL.